ansible/roles/prosody/tasks/install.yml

```diff
@@ -171,6 +171,8 @@
   ansible.posix.patch:
     basedir: /usr/lib/prosody/modules/muc
     src: "muc_lib_visitor_broadcast.patch"
+  when:
+    - ((prosody_minor_version | int) == 12 and (prosody_subversion | int) < 4)
```

Comment:
We're assuming 0.12.4 will contain a fix?

Reply:
Yes, that's what was promised :)